vsintegration/src/vs/FsPkgs/FSharp.VS.FSI/fsiCommands.vsct

```diff
@@ -92,51 +92,36 @@
         </Strings>
       </Button>
 
-      <!-- In Dev11, shell now has 
+      <!-- In Dev11+, shell now has 
       <Button guid ="guidVSStd11" id ="cmdidInteractiveSessionInterrupt" priority ="0x100" type ="Button">
       <Button guid ="guidVSStd11" id ="cmdidInteractiveSessionRestart" priority ="0x105" type ="Button">
       -->
-      <!-- In Dev10 hybrid, we need buttons below -->
```

Comment:
Sorry, a bit of cleanup here (and elsewhere). Wanted to clear out a bunch of VS 2010-era dead code that was also handling "send to FSI" type stuff (but is not longer active).